Lab_1 (1).ipynb

```diff
@@ -0,0 +1,948 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "id": "8dc2a06e-b560-4ecd-bed8-d6fc662e4883",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import scipy\n",
+    "from scipy import stats\n",
+    "import scipy.integrate as integrate\n",
+    "import math\n",
+    "from scipy.stats import lognorm\n",
+    "#this sets the size of the plot to something useful\n",
+    "plt.rcParams[\"figure.figsize\"] = (10,10)\n",
+    "\n",
+    "from IPython.display import Image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73e974a2-aac4-440d-800c-708c89cc9ed5",
+   "metadata": {},
+   "source": [
+    "### A Little Statistics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7e4ce116-f454-436a-9dda-8af066536605",
+   "metadata": {},
+   "source": [
+    "### Problem 1: A normal, or Gaussian, distribution is characterized by a bell curve, where the mean, median, and mode are all the same value in the middle of the distribution and the rest of the distribtion is symmetric around this value. For a standard normal distribution the mean is at 0 and the variance and standard deviatiaon is equal to 1. \n",
+    "Integrating the normal distribution using cdf() will provide us with the probability up to the standard deviation that we choose, the integral of the distribtion is called the error function. Positive sigma values will give us a probability up to the sigma on the right of the mean. Therefore negative sigma values will provide the probability up to the sigma value to the left of the mean. Here are a few probability calculations using stats.norm.cdf() with sigma values 1, 2, and 5 respectively. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "971cda4e-a01f-4865-8b4c-f61e2b4d0215",
+   "metadata": {},
+   "source": [
+    "Probability of sigma = 1, this will give us the percentage probability under the normal distribution if we integrated from negative infinity up to 1 sigma to the right of the mean. Since the distribution is symmetric and centered on 0 sigma, one sigma to the right should correspond to about 84.1%. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "c25f0097-4715-460b-b281-afe3b2654368",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.8413447460685429\n"
```

Comment:
Your outputs should have text labels on them so you don't need to reference the text below to figure out what it is